openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -1430,20 +1430,19 @@ The Wallet is supposed to detect signs of fraudulent behavior related to the Cre
 
 ## Key Proof replay {#key-proof-replay}
 
-If an adversary is able to get hold of a key proof defined in (#proof_types), the adversary could get a Credential issued that is bound to a key pair controlled by the victim.
+If an adversary is able to get ahold of a key proof, as defined in (#proof_types), the adversary could get a Credential issued that is bound to a key pair controlled by the victim.
```

Comment:
can we please keep `hold` instead of `ahold`?

Reply:
It's an awkward area of grammar that differs between USA & British English, and I agree with avoiding 'ahold'. Perhaps just use "obtain" instead?

Reply:
I applied Joseph's proposed wording.